src/components/Button/Button.scss

```diff
@@ -8,7 +8,7 @@
 .button {
   width: fit-content;
   text-align: center;
-  border: none;
+  border: none !important;
```

Comment:
You should use your sass to change this one, like you can target "& button" in the scss